src/HelixProject.js

```diff
@@ -230,6 +235,47 @@ class HelixProject {
     return this.config.strains.get('default');
   }
 
+  matchStrain(request) {
+    // look for X-Strain cookie first
+    if (request.cookies) {
+      const cstrain = this.config.strains.get(request.cookies['X-Strain']);
+      if (cstrain) {
+        return { strain: cstrain };
+      }
+    }
+    const host = request && request.headers ? request.headers.host : '';
+    const reqPath = `${request && request.path && request.path.replace(/\/+$/, '') ? request.path : ''}/`;
+
+    const matches = [...this.config.strains.values()]
+      .map((strain) => ({ strain, mount: null }))
+      .filter((info) => {
+        const { strain } = info;
+        // try selecting strain by condition
+        if (strain.condition) {
+          const result = strain.condition.match(request);
+          if (result.baseURL) {
+            // eslint-disable-next-line no-param-reassign
+            info.mount = result.baseURL;
+          }
+          return !!result;
```

Comment:
`true` ?

Reply:
not quite: `result` is either `false`, `true` or an object containing a property `baseURL`. That statement will only return either `true` or `false`.

Reply:
but if it's `false`, then `if (result.baseURL)` will throw....

Reply:
Nope: running that small test script:
```
'use strict'

var a = false;
if (a.baseURL) {
  console.log('huh?');
} else {
  console.log('that looks right.');
}
```
works as expected.

Reply:
But I can of course change that test to: `if (result && result.baseURL) { ...` to make it clearer.

Reply:
I think `return !!result;` is both idiomatic and concise.

Reply:
> works as expected.

ok...didn't know